lib/gpu/gpuProcess.cpp

```diff
@@ -166,7 +167,7 @@ void gpuProcess::main_thread() {
         DEBUG("Waiting for free slot for GPU[{:d}][{:d}]", gpu_id, gpu_frame_id);
         // We make sure we aren't using a gpu frame that's currently in-flight.
         final_signals[gpu_frame_id]->wait_for_free_slot();
-        queue_commands(gpu_frame_id);
+        queue_commands(gpu_frame_id, gpu_frame_counter);
```

Comment:
Do we want to replace this with a mod counter like `FrameID` now, or address that in another PR?

Reply:
I'd like to hold off on this one, since I'm hoping for more sweeping changes to this in the near future!